common/app/controllers/EmailSignupController.scala

```diff
@@ -39,44 +39,64 @@ case class EmailForm(
     campaignCode: Option[String],
     googleRecaptchaResponse: Option[String],
     name: String,
-) {
-
-  // `name` is a hidden (via css) form input
-  // if it was set to something this form was likely filled by a bot
-  // https://stackoverflow.com/a/34623588/2823715
-  def isLikelyBotSubmission: Boolean =
-    name match {
-      case "" | "undefined" | "null" => false
-      case _                         => true
-    }
-}
+) {}
+
+case class EmailFormManyNewsletters(
+    email: String,
+    listNames: Seq[String],
+    referrer: Option[String],
+    ref: Option[String],
+    refViewId: Option[String],
+    campaignCode: Option[String],
+    googleRecaptchaResponse: Option[String],
+    name: String,
+) {}
 
 class EmailFormService(wsClient: WSClient, emailEmbedAgent: NewsletterSignupAgent)
     extends LazyLogging
     with RemoteAddress {
 
   def submit(form: EmailForm)(implicit request: Request[AnyContent]): Future[WSResponse] = {
-    if (form.isLikelyBotSubmission) {
-      Future.failed(new IllegalAccessException("Form was likely submitted by a bot."))
-    } else {
-      val idAccessClientToken = Configuration.id.apiClientToken
-      val consentMailerUrl = serviceUrl(form, emailEmbedAgent)
-      val consentMailerPayload = JsObject(Json.obj("email" -> form.email, "set-lists" -> List(form.listName)).fields)
-      val headers = clientIp(request)
-        .map(ip => List("X-Forwarded-For" -> ip))
-        .getOrElse(List.empty) :+ "X-GU-ID-Client-Access-Token" -> s"Bearer $idAccessClientToken"
-
-      val queryStringParameters = form.ref.map("ref" -> _).toList ++
-        form.refViewId.map("refViewId" -> _).toList ++
-        form.listName.map("listName" -> _).toList
-
-      //FIXME: this should go via the identity api client / app
-      wsClient
-        .url(consentMailerUrl)
-        .withQueryStringParameters(queryStringParameters: _*)
-        .addHttpHeaders(headers: _*)
-        .post(consentMailerPayload)
-    }
+    val consentMailerUrl = serviceUrl(form, emailEmbedAgent)
+    val consentMailerPayload = JsObject(Json.obj("email" -> form.email, "set-lists" -> List(form.listName)).fields)
+
+    val queryStringParameters = form.ref.map("ref" -> _).toList ++
+      form.refViewId.map("refViewId" -> _).toList ++
+      form.listName.map("listName" -> _).toList
+
+    //FIXME: this should go via the identity api client / app
```

Comment:
Is this a note for this PR or for some future problem to fix?

Reply:
Future problem, I'm afraid. The comment was there before (on a different line). I think it would be a bit beyond me to fix it myself, but happy to pair if it needs doing.